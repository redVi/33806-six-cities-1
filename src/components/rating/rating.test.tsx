import React from 'react';
import renderer from 'react-test-renderer';
import Rating from '@/components/rating/rating';

describe(`Rating`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(<Rating rating={4.6} className="cool" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
