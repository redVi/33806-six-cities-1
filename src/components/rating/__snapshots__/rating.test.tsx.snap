// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating renders correctly 1`] = `
<div
  className="cool__rating rating"
>
  <div
    className="cool__stars rating__stars"
  >
    <span
      style={
        Object {
          "width": "92%",
        }
      }
    />
    <span
      className="visually-hidden"
    >
      Rating
    </span>
  </div>
</div>
`;
